modules/DefaultRenderer.js

```diff
@@ -5,7 +5,7 @@ import React, { Component } from 'react'
 import { NativeModules, StyleSheet, Platform, View } from 'react-native'
 import { Transitioner } from 'react-navigation'
 import Card from 'react-navigation/src/views/Card'
-import CardStackPanResponder from 'react-navigation/src/views/CardStackPanResponder'
+import CardStackStyleInterpolator from 'react-navigation/src/views/CardStackStyleInterpolator'
```

Comment:
Hòla! Fixed in https://github.com/LeoLeBras/react-router-navigation/pull/14/commits/34b92d9b53c28d9722f01a9c9c3c24fbc9bdac52